### ISSUE TITLE
[REC] Outlaw w/ Concealed Blunderbuss recommends pistol shot at max CP with Blunderbuss proc active

### ISSUE THREAD

**Issue by fzarr** [Hekili/hekili#1523]
### Before You Begin

- [X] I confirm that I have downloaded the latest version of the addon.
- [X] I am not playing on a private server.
- [X] I checked for an [existing, open ticket](https://github.com/Hekili/hekili/labels/recommendation) for this issue and was not able to find one.
- [X] I edited the title of this issue (above) so that it describes the issue I am reporting.
- [X] I am reporting an issue with the default priority included with the specialization (imported or edited priorities are not supported).

### Describe the Issue

Expected to use a finisher at max cp (or even max cp -1) first, otherwise over-capping a lot of CP from using the blunderbuss proc.

### How to Reproduce

Difficult to reproduce this one as it doesn't always happen. I followed the rotation on a target dummy until I was able to produce a snapshot of the issue.

### Snapshot (Link)

https://pastebin.com/EJ7eeL6f

**Comment by fzarr** [comment]
I think this might be caused by the finish_condition variable not getting set correctly:

From the snapshot:

`combo_points.current[5.00] >= cp_max_spend[6.00] - buff.broadside.up[false] - ( buff.opportunity.up[true] * talent.quick_draw.enabled[false] | buff.concealed_blunderbuss.up[true] ) | effective_combo_points[5.00] >= cp_max_spend[6.00]` 

The expression `cp_max_spend[6.00] - buff.broadside.up[false] - ( buff.opportunity.up[true] * talent.quick_draw.enabled[false] | buff.concealed_blunderbuss.up[true] )` should be calculated as 5 due to `buff.concealed_blunderbuss.up` being true and subtracting 1, but not sure if that is happening here?

**Comment by Hekili** [comment]
Please provide an updated snapshot using the latest alpha of the addon.

**Comment by elsavira** [comment]
FWIW: quickly checked this out on latest alpha.
https://pastebin.com/8RvxvUEg

**Comment by Hekili** [comment]
Try editing Outlaw - default - 5 from:

```
combo_points >= cp_max_spend - buff.broadside.up - ( buff.opportunity.up * talent.quick_draw.enabled | buff.concealed_blunderbuss.up ) | effective_combo_points >= cp_max_spend
```

To:

```
combo_points >= cp_max_spend - buff.broadside.up - ( buff.opportunity.up & talent.quick_draw.enabled | buff.concealed_blunderbuss.up ) | effective_combo_points >= cp_max_spend
```

(The change is **`*`** to **`&`**.)

**Comment by fzarr** [comment]
Changing * to & seems to have fixed it.

**Comment by elsavira** [comment]
https://pastebin.com/mcDPktd0 to confirm this fix.

**Comment by Hekili** [comment]
Fix is in 1.0.1 (and 1.0.1a).
